src/m_misc.h

```diff
@@ -29,6 +29,8 @@
 #ifdef _WIN32
 wchar_t *M_ConvertUtf8ToWide(const char *str);
 #endif
+char *M_ConvertUtf8ToSysNativeMB(const char *str);
```

Comment:
There should be a comment here that these return a new string which must be `free()`d by the caller.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1557/commits/aca14b69c1e4bc527118ead32a7a5cc6d32334e9